src/patternfly/components/Accordion/examples/accordion-fluid-example.hbs

```diff
@@ -0,0 +1,41 @@
+{{#> accordion}}
+  {{#> accordion-item accordion-item--attribute='aria-expanded="false"'}}
+    {{#> accordion-item-button accordion-item-button--attribute='aria-controls="ex-content-1"'}}
```

Comment:
I know this is an attribute that's suggested in the [wai-aria example for Accordion](https://www.w3.org/TR/wai-aria-practices-1.1/examples/accordion/accordion.html), but I lean toward only using this attribute for cases where the controlled contents do not immediately follow the toggle. I'd suggest to just remove this attribute altogether.